db/dbConnection.js

```diff
@@ -1,13 +1,31 @@
 /* eslint-disable no-undef */
 import pgp from "pg-promise";
 
-const dbConnectionData = {
-  host: process.env.DB_HOST,
-  port: process.env.DB_PORT,
-  database: process.env.DB_DATABASE,
-  user: process.env.DB_USER,
-  password: process.env.DB_PASSWORD,
-};
-const db = pgp()(dbConnectionData);
+let db;
+let environment = false;
+
+process.argv.forEach((element) => {
+  if (element.includes("mocha")) {
+    environment = "test";
+  }
+});
+
+if (environment === "test") {
```

Comment:
Need to be discussed but I think we can use an `.env.test` file here where we put dedicated database information. In this case, we don't need this if.